public/includes/find-house.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sql_query = "SELECT * FROM `states`";
+    $result = mysqli_query($mysqli,$sql_query);
+    $json_response = array();
+    while ($row = mysqli_fetch_assoc($result)) {
+        $json_response[] = $row;
+    }
+    echo json_encode($json_response);
+    db_query_error($sql_query);
+}
+
+function get_area_based_on_state_id($state_id) {
+    $mysqli = db_connection();
+    $state_id = $mysqli->real_escape_string($state_id);
+    $sql_query = "SELECT * FROM `area` WHERE `state_id` = '$state_id'";
+    $result = mysqli_query($mysqli,$sql_query);
+    $json_response = array();
+
+    while ($row = mysqli_fetch_assoc($result)) {
+        $json_response[] = $row;
+    }
+    echo json_encode($json_response);
+    db_query_error($sql_query);
+}
+
+function get_houses_based_on_area_id($state_id, $area_id) {
+    $mysqli = db_connection();
+    $state_id = $mysqli->real_escape_string($state_id);
+    $area_id = $mysqli->real_escape_string($area_id);
+    $sql_query = "SELECT * FROM `house-type` WHERE `area_id` = '$area_id' AND `state_id` = '$state_id'";
```

Comment:
Can we use distinct here to return unique records instead of all the records in the database? Please read up on DISTINCT here - https://www.w3schools.com/sql/sql_distinct.asp.
```suggestion
    $sql_query = "SELECT DISTINCT(type) FROM `house-type` WHERE `area_id` = '$area_id' AND `state_id` = '$state_id'";
```
Returning all the records in the database will inconsistent result like this:
<img width="1375" alt="Screenshot 2020-06-03 at 1 39 09 PM" src="https://user-images.githubusercontent.com/26963369/83639357-7a198800-a5a2-11ea-83a1-8a42219132b9.png">